src/GoTrueClient.ts

```diff
@@ -485,6 +487,7 @@ export default class GoTrueClient {
    * if you are using phone sign in with the 'whatsapp' channel. The whatsapp
    * channel is not supported on other providers
    * at this time.
+   * This method supports PKCE on the magic link flow.
```

Comment:
Wasn't sure if users would understand what "the magic link flow" refers to but couldn't think of a better phrasing offhand. Open to suggestions if there are any.

Reply:
maybe this?

```suggestion
   * This method supports PKCE when an email is passed.
```

Reply:
Looks great, thanks!